### PR TITLE
Update French translation 

### PR DESCRIPTION
**Pull request by scandella**
Some additions and fixes to the French translation.

### CODE REVIEW

File: src/translations/fr/campaign.php [hunk 1]
```diff
@@ -289,37 +289,37 @@
...
     'Imported By' => 'Importer par',
     'Importing contacts.' => 'Importer des contacts',
     'Initialising plugin' => 'Initialisation du plugin',
     'Inline' => 'En ligne',
     'Interaction' => '',
     'interactions' => '',
-    'Invalid verification link.' => '',
+    'Invalid verification link.' => 'Lien de vérification invalide',
     'Invisible' => '',
-    'IP address not allowed.' => '',
```
Comment (bencroker) [inline src/translations/fr/campaign.php:313]: Please restore this.

File: src/translations/fr/campaign.php [hunk 2]
```diff
@@ -289,37 +289,37 @@
...
-    'ipstack.com API Key' => '',
+    'Invisible' => 'Interaction',
```
Comment (bencroker) [inline src/translations/fr/campaign.php]: Please double check if this is the correct translation.

File: src/translations/fr/campaign.php [hunk 3]
```diff
@@ -406,7 +406,7 @@
     'No sendouts have been sent to this mailing list.' => 'Aucune campagne n’a été envoyée à cette liste de diffusion.',
     'No synced mailing lists exist.' => '',
     'No user groups exist.' => 'Aucun groupe d’utilisateurs existant.',
-    'None' => 'Auncun',
+    'none' => 'Non inscrit',
```
Comment (bencroker) [inline src/translations/fr/campaign.php]: Please restore the key to be capitalised (`None`) as translations are case-sensitive.

**Review by bencroker — CHANGES_REQUESTED** [review]
Thanks for the PR @scandella! I've spotted a few things which I'd like you to look at and resolve before merging.

**Comment by scandella** [comment]
Hi @bencroker ,
I've made some changes to the translation, according to your remarks. Sorry for the confusion…

Best,

**Comment by bencroker** [comment]
Merged, thank you!

**Pull request merged**